src/SFA.DAS.EmployerCommitmentsV2.Web/Views/CreateCohort/ConfirmProvider.cshtml

```diff
@@ -1,7 +1,57 @@
-﻿
+﻿@using SFA.DAS.EmployerCommitmentsV2.Web.Models.CreateCohort
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.CreateCohort.ConfirmProviderViewModel
```

Comment:
could we drop the namespace on the model? As it's in the 'using' line

Reply:
done.